packages/go_router_builder/_go_router_fake/lib/go_router.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+export 'src/route_data.dart';
+
+abstract class BuildContext {
```

Comment:
I 100% agree. Sadly we're limited by `pkg:analyzer` and the build system.

Testing will fail (I think) because we have build tests that verify the builder still works. See the `ensure_build_test.dart` files